src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -988,7 +988,7 @@ public override IEnumerable<IOperation> Children
         {
             get
             {
-                yield break;
+                return Array.Empty<IOperation>();
```

Comment:
:bulb: `Array.Empty<T>` doesn't have a non-allocating implementation of `GetEnumerator()`, but Roslyn's specialized `Empty.List<T>` does.